creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -448,7 +456,7 @@ async function filterOutAlreadyPresentDBEntries({
   return filteredEntries
 }
 
-async function getUserReplicaSet({ wallet, selfEndpoint, libs, logger }) {
+async function getUserReplicaSet({ wallet, libs, logger }) {
   try {
     let userReplicaSet = await getCreatorNodeEndpoints({
```

Comment:
we probably shouldn't be using this function since it polls discovery. preferably we use https://github.com/AudiusProject/audius-protocol/blob/bc0256a9bce806bc5dfe58ffd2d00c496e20ec80/creator-node/src/middlewares.js#L717. we'd have to do a lookup for spID to endpoint (or use this https://github.com/AudiusProject/audius-protocol/blob/788d4821547b2f4824a369759cf97df4e43fa75b/creator-node/src/services/stateMachineManager/ContentNodeInfoManager.js#L22) but this seems much safer

Reply:
will handle in followup PR